Dockerfile

```diff
@@ -19,9 +17,18 @@ RUN pip3 install --upgrade pip && pip install --upgrade cmake && \
   cmake --version
 
 # VAST
-RUN git clone --recursive https://github.com/tenzir/vast $BUILD_DIR/vast
 WORKDIR $BUILD_DIR/vast
-RUN git checkout ${BRANCH}
+COPY aux ./aux
+COPY cmake ./cmake
+COPY doc ./doc
+COPY integration ./integration
+COPY libvast ./libvast
+COPY libvast_test ./libvast_test
+COPY schema ./schema
+COPY scripts ./scripts
+COPY tools ./tools
+COPY vast ./vast
+COPY .clang-format .cmake-format BANNER CMakeLists.txt configure vast.conf ./
```

Comment:
This seems brittle, if we must keep it in sync with the repository layout manually.

Reply:
That is just the way it is with Docker. The alternative is to clone the repository, which we did before. I moved away from that since it is very error prone wrt Docker caching.

Reply:
Is there really no way to just copy the whole directory, something like `COPY . ./`? Or even just the output of `git ls-files`.

Reply:
Copying the whole directory invalidates the benefit you gain from selective copies. We could do that but would have to maintain a `.dockerignore` file then, which prevents unnecessary files from copying.

Reply:
I'll post a poll in Slack.

Reply:
Resolving as per discussion in Slack.